README.md

```diff
@@ -102,156 +103,127 @@ There are [community driven install methods](./doc/unofficialInstallMethods.md)
- - New dev docs for the Light Switch module.
-
-### Development (Area-Build & Area-Tests) 
-- Allowed debug launches to continue when modules fail to load, speeding developer iteration. 
-- Fixed spell checker dictionary entry (advapi) to eliminate false error. 
-- Added VS Code development guide and launch configs to streamline cross-editor workflows. 
-- Upgraded Windows App SDK and related dependencies to 1.8 for newer platform features. 
-- Rewrote YAML comment to resolve new spell checker forbidden pattern. Thanks [@jiripolasek](https://github.com/jiripolasek)! 
-- Corrected solution structure by returning misplaced Common project, reducing build confusion. 
-- Modernized build scripts with shared helpers and VS environment autodetection for simpler CLI builds. 
-- Standardized build scripts and platform detection to improve reliability and reuse. 
-- Added missing Command Palette version bump to align module release cadence. 
-- Added EXECUTEDEFAULT term to dictionary to prevent regression build failures. Thanks [@jiripolasek](https://github.com/jiripolasek)! 
-- Introduced nightly pre-warm pipeline and configurable MSBuild cache mode to improve CI performance. 
-- Resolved CI forbidden pattern spelling complaint to keep pipelines green. 
-- Added AI contributor instruction set to clarify code area expectations. 
-- Added accessibility IDs to settings and FancyZones toggles, stabilizing UI tests. 
-- Added automatic log collection on UI test failures to speed root cause analysis. 
-- Stabilized Mouse Utils tests by switching to AccessibilityId selectors. 
-- Added Screen Ruler UI test coverage to validate core measurement workflows. 
+- Fixed title bar overlapping issue at smaller window sizes.
+- Refined shortcut control visual design with improved consistency and spacing.
+- Added dashboard utilities sorting by name or status.
+- Made update notification InfoBar in flyout clickable for direct navigation to update page.
+- Expanded installation instructions by default in README.
+- Improved accessibility for shortcut conflict button with static resource-based automation properties.
+
+### Development
+- Fixed accessibility by associating controls with labels for screen readers.
+- Added accessible name to Shortcut Conflicts button for screen readers.
+- Excluded TitleBars from tab navigation across multiple utilities. Thanks [@jiripolasek](https://github.com/jiripolasek)!
+- Migrated build infrastructure from Windows Server 2019 to Server 2022 with improved failure logging and predictable NuGet package paths.
+- Configured build agents to use larger P: drive for release builds to address disk space constraints.
+- Enhanced DSC v3 support by organizing resource manifests in a dedicated subfolder with PATH configuration.
+- Reduced installer bundle size by 6-7MB through centralized Hybrid CRT configuration across all C++ projects.
+- Updated .NET packages to version 9.0.10 for security fixes. Thanks [@snickler](https://github.com/snickler)!
```

Comment:
[nit] For the "Thanks" messages, we should standardise on whether they should end in exclamation marks or not. There are inconsistencies throughout.